qcodes/utils/oscilloscope.py

```diff
@@ -32,6 +33,7 @@ class Oscilloscope(ParameterNode):
     # Define settings
     oscilloscope.ylim = (-0.8, 1)
     oscilloscope.channels_settings['chB']['scale'] = 10
+    oscilloscope.channels_settings['chB']['name'] = 'ESR'
```

Comment:
why is it specific to ESR?

Reply:
It's just an example, no specific meaning.

Reply:
It's just an example, no specific meaning.

Reply:
So I've managed to fixed the issues with updating the legend, give it a go on your measurement PC when you get the chance!

Reply:
great!